server/server_test.go

```diff
@@ -0,0 +1,677 @@
+package server_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/cosmos/iavl"
+	"github.com/stretchr/testify/suite"
+	dbm "github.com/tendermint/tm-db"
+
+	pb "github.com/cosmos/iavl/proto"
+	"github.com/cosmos/iavl/server"
+)
+
+type ServerTestSuite struct {
+	suite.Suite
+
+	server *server.IAVLServer
+}
+
+func (suite *ServerTestSuite) SetupTest() {
+	db, err := dbm.NewDB("test", dbm.MemDBBackend, "")
+
+	suite.NoError(err)
+
+	server, err := server.New(db, 1000, 0)
+	suite.NoError(err)
+
+	suite.server = server
+	suite.populateItems(100)
+}
+
+func (suite *ServerTestSuite) populateItems(n int) {
+	versionRes, err := suite.server.Version(context.TODO(), nil)
```

Comment:
Use `context.Background()`, not `context.TODO()` - the latter is only meant to be used as a placeholder/marker while porting non-context-aware code to newer context APIs, and should not be used in finished code.